### PR TITLE
Feature: Attachments on Send Email

### PR DESCRIPTION
**Pull request by ihor-palii**
Modified Legacy Flow Editor to accept attachments on Send Email.

Details:
Front: https://app.asana.com/0/1159876841343611/1158544836278234/f
Back: https://app.asana.com/0/1159876841343611/1158544836278232/f

### CODE REVIEW

File: static/coffee/flows/controllers.coffee [hunk 1]
```diff
@@ -212,7 +212,7 @@ app.controller 'FlowController', [ '$scope', '$rootScope', '$timeout', '$log', '
         showDialog('Invalid Format', 'Audio attachments must be encoded as mp3, m4a, wav, ogg or oga files.')
         return
 
-    if action.type in ['reply', 'send'] and (file.size > 20000000 or (file.name.endsWith('.jpg') and file.size > 500000))
+    if action.type in ['reply', 'send', 'email'] and (file.size > 20000000 or (file.name.endsWith('.jpg') and file.size > 500000))
       showDialog('File Size Exceeded', "The file size should be less than 500kB for images and less than 20MB for audio and video files. Please choose another file and try again.")
       return
```
Comment (teehamaral) [inline static/coffee/flows/controllers.coffee:218]: Attachments on send email should not allow those type of files, and allow all the rest:
https://github.com/greatnonprofits-nfp/rapidpro/blob/2fed4ba886867143ad80d79da7f3bffd08af6ad1/static/coffee/flows/controllers.coffee#L215

File: templates/partials/action_editor.haml [hunk 2]
```diff
@@ -212,8 +212,40 @@
       .label
         -trans "Message"
       %textarea{ng-model:"action.msg",auto-complete:"",flow:"[[flowId]]",name:"body",ng-required:"true"}
```
[the remark this answers is not in the document]
Comment (ihor-palii) [reply templates/partials/action_editor.haml]: Have added styles.

File: static/coffee/flows/controllers.coffee (hunk 1 above)
Comment (ihor-palii) [reply static/coffee/flows/controllers.coffee:218]: Have added type validation and removed validation by size for the email attachment. Or size validation should be?

Comment (teehamaral) [reply static/coffee/flows/controllers.coffee:218]: The size I would say that's required under 25MB

Comment (ihor-palii) [reply static/coffee/flows/controllers.coffee:218]: Have set limit to 25MB for email attachments size.

**Pull request merged**